### ISSUE TITLE
Provide extra data with scores

### ISSUE THREAD

**Issue by Hornwitser** [explosivegaming/factorio-olympics#32]
**Feature overview**
Name: extra-score-data
TL;DR: Give more information about the type of game and extra points of scores in matches.

**Details**
Currently only a single number "score" and the players that got that score is provided back to the airtable.  But for the data to be useful we need to at least know the variation of the game that was run.  For the race game that is the number of laps, the fuel type, and map the race was held on.  Possibly also the lap times.

**Possible methods**
Return the data via the stopped_game write.

**Additional context**
Must also have a corresponding implementation in http://github.com/tovernaar123/Airtable

**Comment by Cooldude2606** [comment]
Pr has been merged, closing this issue.
